ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/intended/configs/SITE1-LER1.cfg

```diff
@@ -146,6 +146,8 @@ interface Ethernet4
 interface Ethernet6
    no shutdown
    no switchport
+   no lldp transmit
+   no lldp receive
```

Comment:
These changes are actually a fix caused by the same interface being auto-created as parent for subinterfaces as well as being used as endpoint for a pseudowire.
The new code will not create a parent interface if it was already created by something else.